packages/ihe-gateway-sdk/src/models/shared.ts

```diff
@@ -0,0 +1,5 @@
+import { z } from "zod";
+
+export const npiStringSchema = z.string().regex(/^\d+$/, "NPI strings must consist of numbers");
```

Comment:
you might want to use the validateNPI function defined in the commonwell-sdk to confirm the NPI is valid?

Reply:
💯 thx for reminding